File: NAFServer/src/Domain/Entities/Employee.cs
namespace NAFServer.src.Domain.Entities
{
    public class Employee
    {
        public string Id { get; set; }
        public string? FirstName { get; set; }
        public string? MiddleName { get; set; }
        public string? LastName { get; set; }
        public string? FullName { get; set; }
        public string? Status { get; set; }
        public string? Company { get; set; }
        public string? Position { get; set; }
        public string? Location { get; set; }
        public string? SupervisorId { get; set; }
        public string? DepartmentId { get; set; }
        public string? DepartmentDesc { get; set; }
        public string? DepartmentHead { get; set; }
    }
}

File: NAFClient/src/shared/types/api/employee.ts
export type Employee = {
  id: string;
  firstName: string;
  middleName?: string;
  lastName: string;
  fullName: string;
  status: string;
  company?: string;
  position?: string;
  location?: string;
  supervisorId?: string;
  departmentId: string;
  departmentDesc: string;
  departmentHead: string;
};
